redhat-developer/odo
# Initial implementation for service create interactive mode.

What is the purpose of this change? What does it change?
Add an interactive mode to `service create` so that users are prompted for needed values if they are not provided with flags.

Was the change discussed in an issue?
Adding an interactive mode to `odo` was discussed in https://github.com/redhat-developer/odo/issues/76 but this change is just a tiny part of this overall effort.

How to test changes?

Invoke `odo service create` with any combination of arguments and flags and see what happens!
